site/src/routes/repl/[id]/_components/AppControls/index.svelte

```diff
@@ -62,6 +63,9 @@
 			const gist = await r.json();
 			dispatch('forked', { gist });
 
+			modified = 0
```

Comment:
i wonder why `modified` is a number instead of boolean

Reply:
good point, it should be a boolean, updating now